### PR TITLE
[stable/redis] Add support for custom labels on redis services

### PR DESCRIPTION
**Pull request by bderickson**
The various redis services already supported custom annotations, but didn't allow for custom labels.

### CODE REVIEW

File: stable/redis/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: redis
-version: 9.2.2
+version: 9.2.3
```
Comment (miguelaeh) [inline stable/redis/Chart.yaml]: Please bump the minor version as you are adding a new feature.

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thank you very much for the PR!!!
Please, add the new parameter to the `README.md`.

File: stable/redis/Chart.yaml (hunk 1 above)
Comment (bderickson) [reply stable/redis/Chart.yaml]: Done! Had to force push because I forgot to sign a couple of my commits and had to alter the commit messages.

**Comment by bderickson** [comment]
I've made all the updates you requested, please re-review. Thanks!

File: stable/redis/templates/redis-with-sentinel-svc.yaml [hunk 2]
```diff
@@ -8,6 +8,9 @@ metadata:
     chart: {{ template "redis.chart" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+    {{- if .Values.master.service.labels }}
```
Comment (miguelaeh) [inline stable/redis/templates/redis-with-sentinel-svc.yaml]: This should be the sentinel labels not the master.

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thank you very much for addressing the changes , please take a look this one too.

File: stable/redis/templates/redis-with-sentinel-svc.yaml (hunk 2 above)
Comment (bderickson) [reply stable/redis/templates/redis-with-sentinel-svc.yaml]: Good catch! Fixed.

**Review by miguelaeh — APPROVED** [review]
Thanks!!!!

**Pull request merged**